build.gradle

```diff
@@ -90,15 +93,6 @@ tasks.register('copyBallerinaClassFiles') {
                 jarFiles.forEach { jar ->
                     zipTree(jar).matching {
                         exclude '**/tests/*'
-                        exclude '**/$_init.class'
-                        exclude '**/$value$Caller.class'
-                        exclude '**/$value$Detail.class'
-                        exclude '**/$value$DetailType.class'
-                        exclude '**/$value$EmptyIterator.class'
-                        exclude '**/$value$$anonType$_6.class'
-                        exclude '**/$value$$anonType$_*.class'
-                        exclude '**/$value$_Frame.class'
-                        include '**/*.class'
```

Comment:
We need to create a classFileArray to map the class files which have the java sources to mark the line by line coverage on the source files. For that shouldn't we exclude the generated class files?

Reply:
Shouldn't generated class files should be covered as well?